### PR TITLE
:bug: igonre failed to get pullsecret error

### PR DESCRIPTION
**Pull request by zhiweiyin318**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ func (n *clusterManagerController) sync(ctx context.Context, controllerContext f
 	// check imagePulSecret here because there will be a warning event FailedToRetrieveImagePullSecret
 	// if imagePullSecret does not exist.
 	if config.ImagePullSecret, err = n.getImagePullSecret(ctx); err != nil {
-		return err
+		klog.Errorf("failed to get image pull secret: %v", err)
```
Comment (zhujian7) [inline pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go:265]: if this error will not result in running to fail, it should be Info or Warning level.

Comment (qiujian16) [reply pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go:265]: ditto, also add a todo here that we should return err in the next release.

Comment (zhiweiyin318) [reply pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go]: done

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**